src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,202 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String createRequest = "INSERT INTO cars(manufacturer_id, model) VALUES(?, ?);";
+        try (Connection dbConnection = ConnectionUtil.getConnection();
+                PreparedStatement createStatement = dbConnection.prepareStatement(createRequest,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setLong(1, car.getManufacturer().getId());
+            createStatement.setString(2, car.getModel());
+            createStatement.executeUpdate();
+            ResultSet generatedKeys = createStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                car.setId(generatedKeys.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert car: " + car
+                    + " to DB", e);
+        }
+        insertDrivers(car);
```

Comment:
Do not implement following methods in DAO layer, only in service layer. Use update method from DAO.

void addDriverToCar(Driver driver, Car car);
void removeDriverFromCar(Driver driver, Car car);

see https://mate-academy.github.io/jv-program-common-mistakes/java-JDBC/join/Joins_checklist.html